### DISCUSSION TITLE
Add "groove mode" mod for osu! mania

### DISCUSSION THREAD

**Discussion by Elkunz** [ppy/osu#26485] [Ideas]
I love to play in osu! mania mode but I have idea for more fun with this mod (Be like BMS / arcaea or osu! taiko mode)

Settings: HP set to 8 or 8.5 , target health to pass (Default):70%
Multiplier:1.0x 
unrank (or maybe rank ?)

When using this Mod
Health bar start at 0% and not fail when health bar got to 0%
If health bar reach 70% or more at the end of song it's song pass / completed
but health bar below 70% at the end of song it's song fail

**Comment by nagi-desuuu** [comment]
Basically taking the HP system from Taiko and putting it in mania as a mod?

**Reply by Elkunz** [reply]
Yes
But taiko mode set a hp target at 50% to complete right?
